grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -1049,6 +1049,51 @@ class StockMovementService {
         }
     }
 
+    void updateAdjustedItems(StockMovement stockMovement) {
+        if (stockMovement?.pickPage?.pickPageItems) {
```

Comment:
you could use this list here and find only those stock movement items that uses the adjusted products